### ISSUE TITLE
Improve build system

### ISSUE THREAD

**Issue by cereallarceny** [OpenMined/syft.js#30]
We need to allow for the Syft.js build system to be compiled and usable (via NPM install).  Currently the package is published here: https://www.npmjs.com/package/syft.js. We need to be able to include it in the following formats:

## Methods of importing
**ES6/7 import:**
```js
import syft from 'syft.js';
```

**Require:**
```js
const syft = require('syft.js');
```

**Script include:**
```html
<script src="/node_modules/syft.js/lib/index.js"></script>
```

## Referencing in Javascript
```js
const mySyft = new syft(url);
```

**Comment by k105la** [comment]
I can work on this

**Comment by cereallarceny** [comment]
All yours.  Would love to have a proof of concept in the examples too if you wouldn't mind!  Not necessary, but would be nice.  Came across this if it helps: https://medium.com/@kelin2025/so-you-wanna-use-es6-modules-714f48b3a953

**Comment by cereallarceny** [comment]
Closed by #35
